RecoLocalCalo/HcalRecProducers/plugins/alpaka/Mahi.h

```diff
@@ -0,0 +1,54 @@
+
+#include <vector>
+
+#include "DataFormats/HcalDigi/interface/alpaka/HcalDigiDeviceCollection.h"
+#include "DataFormats/HcalRecHit/interface/alpaka/HcalRecHitDeviceCollection.h"
+#include "CondFormats/HcalObjects/interface/alpaka/HcalMahiConditionsDevice.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/traits.h"
+#include "CondFormats/HcalObjects/interface/alpaka/HcalMahiConditionsDevice.h"
+#include "CondFormats/HcalObjects/interface/alpaka/HcalSiPMCharacteristicsDevice.h"
+#include "CondFormats/HcalObjects/interface/alpaka/HcalRecoParamWithPulseShapeDevice.h"
+#include "CondFormats/HcalObjects/interface/alpaka/HcalMahiPulseOffsetsDevice.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE::hcal::reconstruction {
+
+  struct ConfigParameters {
+    uint32_t maxTimeSamples;
+    uint32_t kprep1dChannelsPerBlock;
+    int sipmQTSShift;
+    int sipmQNTStoSum;
+    int firstSampleShift;
+    bool useEffectivePedestals;
+
+    float meanTime;
+    float timeSigmaSiPM, timeSigmaHPD;
+    float ts4Thresh;
+
+    std::array<uint32_t, 3> kernelMinimizeThreads;
+
+    // FIXME:
+    //   - add "getters" to HcalTimeSlew calib formats
+    //   - add ES Producer to consume what is produced above not to replicate.
+    //   which ones to use is hardcoded, therefore no need to send those to the device
+    bool applyTimeSlew;
+    float tzeroTimeSlew, slopeTimeSlew, tmaxTimeSlew;
+  };
+
+  using IProductTypef01 = hcal::Phase1DigiDeviceCollection;
+  using IProductTypef5 = hcal::Phase0DigiDeviceCollection;
+  using IProductTypef3 = hcal::Phase1DigiDeviceCollection;
+  using OProductType = hcal::RecHitDeviceCollection;
+
+  void entryPoint(Queue& queue,
```

Comment:
Could you rename `entryPoint` to something more descriptive, like `enqueueMahi` or `runMahiAsync` or something like that ?

Reply:
Updated in 715105876f8999402d3e30f2fa94579275479978